# There are some strange shells in the wild, but not on OpenBSD.
# Removing the tab from the closing line isn't an option since this
# would be unethical.

--- autoopts/test/enums.test.orig	Sat Sep 23 03:45:13 2006
+++ autoopts/test/enums.test	Fri Nov 17 20:04:44 2006
@@ -190,7 +190,7 @@ cat > ${testname}.tpl <<- \__EOF
 	}
 	__EOF
 
-cat >> ${testname}.tpl <<__EOF
+cat >> ${testname}.tpl <<- __EOF
 	int
 	main( int argc, char** argv )
 	{
